canvas_modules/common-canvas/src/common-properties/controls/list/list.jsx

```diff
@@ -158,19 +159,23 @@ class ListControl extends AbstractTable {
 			</div>
 			<ValidationMessage state={this.props.state} messageInfo={this.props.messageInfo} />
 		</div>);
+		const disabled = this.props.state === STATES.DISABLED;
+		const addRemoveRows = has(this.props.control, "addRemoveRows") ? this.props.control.addRemoveRows : true;
 
 		return (
 			<div data-id={getDataId(this.props.propertyId)} className="properties-list-control" >
 				{this.props.controlItem}
-				<MoveableTableRows
-					tableContainer={tableContainer}
-					control={this.props.control}
-					controller={this.props.controller}
-					propertyId={this.props.propertyId}
-					setScrollToRow={this.setScrollToRow}
-					setCurrentControlValueSelected={this.setCurrentControlValueSelected}
-					disabled={this.props.state === STATES.DISABLED}
-				/>
+				{isEmpty(this.props.value) && addRemoveRows ? this.makeEmptyTablePanel(tableButtonConfig.addButtonLabel, this.addRow, disabled)
```

Comment:
Any reason not to put this logic all in MoveableTableRows?  Then each component doesn't need to have this check.